### DISCUSSION TITLE
反复重启，bug怎么没修复，一个多月了

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#169] [Bugs/问题反馈]
刷入最新版本编译的固件，直接反复重启，一直死循环，没有日志
我刷了uboot,不然路由器就挂了

**Comment by kiddin9** [comment]
回退到 https://github.com/Boos4721/openwrt/tree/0890af20fadfd30b9e201fc7c279cdcabf2120f1 后正常.
